### PR TITLE
pt-BR: cr2 concurrency and glossary translations

### PR DESCRIPTION
**Pull request by henrif75**
#317 #1463

### CODE REVIEW

File: po/pt-BR.po [hunk 1]
```diff
@@ -21217,12 +21411,19 @@ msgid ""
 "blog/2019/10/26/async-fn-in-traits-are-hard/) if you are interested in "
 "digging deeper."
 msgstr ""
+"Os desafios no suporte da linguagem para `async trait` são profundos em Rust "
+"e provavelmente não valem a pena descrever em detalhes. Niko Matsakis fez um "
+"bom trabalho ao explicá-los em [este post](https://smallcultfollowing.com/"
```
Comment (joaovicmendes) [inline po/pt-BR.po:21416]: nit:
```suggestion
"bom trabalho ao explicá-los [neste post](https://smallcultfollowing.com/"
```

File: po/pt-BR.po [hunk 2]
```diff
@@ -21231,126 +21432,156 @@ msgid ""
...
 
 #: src/async/pitfalls/cancellation.md:35
 msgid "\"not UTF-8\""
-msgstr ""
+msgstr "\"não UTF-8\""
 
 #: src/async/pitfalls/cancellation.md:51
 msgid "\"hi\\nthere\\n\""
-msgstr ""
+msgstr "\"oi\\nali\\n\""
```
Comment (joaovicmendes) [inline po/pt-BR.po:21447]: [q]: is the original "hi there" meaning hello, or is "there" meaning "that place"?

File: po/pt-BR.po [hunk 3]
```diff
@@ -21231,126 +21432,156 @@ msgid ""
...
 msgstr ""
+"Para praticar suas habilidades em Rust Async, temos novamente dois "
+"exercícios para você:"
 
 #: src/exercises/concurrency/afternoon.md:5
 msgid ""
 "Dining philosophers: we already saw this problem in the morning. This time "
 "you are going to implement it with Async Rust."
 msgstr ""
+"Filósofos jantando: já vimos esse problema de manhã. Desta vez, você vai "
```
Comment (joaovicmendes) [inline po/pt-BR.po:21536]: Same as before.
```suggestion
"Jantar dos Filósofos: já vimos esse problema de manhã. Desta vez, você vai "
```

File: po/pt-BR.po [hunk 4]
```diff
@@ -21231,126 +21432,156 @@ msgid ""
...
 msgstr ""
+"Um aplicativo de bate-papo por _broadcast_: este é um projeto maior que "
+"permite experimentar com recursos mais avançados de Rust Async."
 
 #: src/exercises/concurrency/dining-philosophers-async.md:1
 #: src/exercises/concurrency/solutions-afternoon.md:3
-#, fuzzy
 msgid "Dining Philosophers --- Async"
-msgstr "Filósofos jantando"
+msgstr "Filósofos jantando --- Async"
```
Comment (joaovicmendes) [inline po/pt-BR.po:21550]:
```suggestion
msgstr "Jantar dos Filósofos --- Async"
```

File: po/pt-BR.po [hunk 5]
```diff
@@ -21231,126 +21432,156 @@ msgid ""
...
 
 #: src/exercises/concurrency/dining-philosophers-async.md:3
 msgid ""
 "See [dining philosophers](dining-philosophers.md) for a description of the "
 "problem."
 msgstr ""
+"Veja [filósofos jantando](dining-philosophers.md) para uma descrição do "
```
Comment (joaovicmendes) [inline po/pt-BR.po:21557]:
```suggestion
"Veja [Jantar dos Filósofos](dining-philosophers.md) para uma descrição do "
```

File: po/pt-BR.po [hunk 6]
```diff
@@ -22203,88 +22672,85 @@ msgid "A small selection of other guides and tutorial for Rust:"
...
+"diferentes de estruturas de lista."
 
 #: src/other-resources.md:61
 msgid ""
 "Please see the [Little Book of Rust Books](https://lborb.github.io/book/) "
 "for even more Rust books."
 msgstr ""
 "Consulte o [Little Book of Rust Books](https://lborb.github.io/book/) para "
-"ainda mais livros Rust."
+"ainda mais livros em Rust."
```
Comment (joaovicmendes) [inline po/pt-BR.po:22753]:
```suggestion
"ainda mais livros sobre Rust."
```

**Review by joaovicmendes — APPROVED** [review]

File: po/pt-BR.po (hunk 2 above)
Comment (henrif75) [reply po/pt-BR.po:21447]: Since this string is embedded in the code, let's keep it original.

**Comment by henrif75** [comment]
@joaovicmendes That was fast, thank you!

**Pull request merged**